### DISCUSSION TITLE
请问在MacOS上怎么部署运行服务呢

### DISCUSSION THREAD

**Discussion by Ssiswent** [MetaCubeX/mihomo#904] [Q&A]
Wiki上似乎只有Linux系统的，请问有mac的教程吗，感谢！
https://wiki.metacubex.one/startup/service/

**Comment by RealTong** [comment]
1. 将 plist 添加到 `/Library/LaunchDaemons`
https://github.com/RealTong/Profile/tree/main/Applications/Clash-Meta(mihomo)
2. `sudo launchctl start macos.meta.service # 启动服务`

还可以根据 https://github.com/RealTong/Profile/blob/6f496ac6e511f59fa460c02ed69c423ca0bac236/Platform/macOS/.zshrc#L114 设置 alias 缩短命令
